src/map/script.cpp

```diff
@@ -26038,6 +26038,36 @@ BUILDIN_FUNC( enchantgradeui ){
 #endif
 }
 
+BUILDIN_FUNC(item_enchant){
+#if PACKETVER_RE_NUM < 20211103
+	ShowError( "buildin_item_enchant: This command requires packet version 2021-11-03 or newer.\n" );
+	return SCRIPT_CMD_FAILURE;
+#else
+	struct map_session_data* sd;
+
+	if( !script_charid2sd( 3, sd ) ){
+		return SCRIPT_CMD_FAILURE;
+	}
+
+	// Hardcoded clientside check
+	if( sd->weight >= ( ( sd->max_weight * 70 ) / 100 ) ){
+		ShowError( "buildin_item_enchant: Player %s (AID: %u, CID: %u) was over 70% weight.\n", sd->status.name, sd->status.account_id, sd->status.char_id );
+		return SCRIPT_CMD_FAILURE;
+	}
```

Comment:
show error to client

https://github.com/llchrisll/ROenglishRE/blob/master/Renewal/data/msgstringtable.txt#L3838

Reply:
No need to trigger this manually. As the comment says, it is hardcoded clientside and will trigger that message.